core/server/api_container/server/service_network/default_service_network.go

```diff
@@ -56,6 +58,12 @@ const (
 	exactlyOneShortenedUuidMatch = 1
 
 	singleServiceStartupBatch = 1
+
+	waitForPortsOpenRetriesDelayMilliseconds = uint16(100)
```

Comment:
100 millis?? Is that a sane default? It's like 10 pings per seconds, seems a lot to me, but why not

Reply:
you probably right, I will increase it to 500 milliseconds, just 2 pings by second